l1-contracts/contracts/bridge/ntv/L1NativeTokenVault.sol

```diff
@@ -93,11 +93,11 @@ contract L1NativeTokenVault is IL1NativeTokenVault, IL1AssetHandler, NativeToken
             }
         } else {
             uint256 balanceBefore = IERC20(_token).balanceOf(address(this));
-            uint256 sharedBridgeChainBalance = IERC20(_token).balanceOf(address(ASSET_ROUTER));
-            require(sharedBridgeChainBalance > 0, "NTV: 0 amount to transfer");
+            uint256 nullifierChainBalance = IERC20(_token).balanceOf(address(L1_NULLIFIER));
```

Comment:
does it make sense to rename the function too? Or maybe even better just add a comment that what used to be "shared bridge" now is the "L1Nullfier"